lib/rgl/base.rb

```diff
@@ -329,4 +329,22 @@ def each_edge_aux
 
   end # module Graph
 
+  def set_vertex_options(vertex, **options)
```

Comment:
The `set_*_options` functions should be moved to `dot.rb`. You can extend module `Graph` there too. See `vertex_label` there.